src/styles/index.scss

```diff
@@ -1,3 +1,125 @@
+@import "/src/styles/variables";
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
 }
+
+.card {
+  display: flex;
+  width: 198px;
+  height: 406px;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+
+  border: 1px solid #F3F3F3;
+  border-radius: 5px;
+
+  &__photo {
+    width: 160px;
+    height: 134px;
+    margin-top: 16px;
+    flex-shrink: 0;
+    background-image: url(../images/imac.jpeg);
+    background-size: 100% 100%;
+    background-repeat: no-repeat;
+    background-position: center;
+  }
+
+  &__title {
+    width: 166px;
+    margin: 0;
+    padding: 40px 0 4px;
+    color: var(--main-accent);
+    font-size: 12px;
+    font-style: normal;
```

Comment:
this is a default browser value, no needs to redeclare it. remove from other cases as well
```suggestion
```